deploy/olm-catalog/toolchain-host-operator/0.0.1/toolchain_v1alpha1_nstemplatetier_crd.yaml

```diff
@@ -41,233 +41,23 @@ spec:
-                            type: string
-                          displayName:
-                            description: 'Optional: The name that will show in UI
-                              instead of parameter ''Name'''
-                            type: string
-                          from:
-                            description: From is an input value for the generator.
-                              Optional.
-                            type: string
-                          generate:
-                            description: "generate specifies the generator to be used
-                              to generate random string from an input value specified
-                              by From field. The result string is stored into Value
-                              field. If empty, no generator is being used, leaving
-                              the result Value untouched. Optional. \n The only supported
-                              generator is \"expression\", which accepts a \"from\"
-                              value in the form of a simple regular expression containing
-                              the range expression \"[a-zA-Z0-9]\", and the length
-                              expression \"a{length}\". \n Examples: \n from             |
-                              value ----------------------------- \"test[0-9]{1}x\"
-                              \ | \"test7x\" \"[0-1]{8}\"       | \"01001100\" \"0x[A-F0-9]{4}\"
-                              \ | \"0xB3AF\" \"[a-zA-Z0-9]{8}\" | \"hW4yQU5i\""
-                            type: string
-                          name:
-                            description: Name must be set and it can be referenced
-                              in Template Items using ${PARAMETER_NAME}. Required.
-                            type: string
-                          required:
-                            description: 'Optional: Indicates the parameter must have
-                              a value.  Defaults to false.'
-                            type: boolean
-                          value:
-                            description: Value holds the Parameter data. If specified,
-                              the generator will be ignored. The value replaces all
-                              occurrences of the Parameter ${Name} expression during
-                              the Template to Config transformation. Optional.
-                            type: string
-                        required:
-                        - name
-                        type: object
-                      type: array
-                  required:
-                  - objects
-                  type: object
                 templateRef:
                   description: TemplateRef The name of the TierTemplate resource which
                     exists in the host cluster and which contains the template to
                     use
                   type: string
-              required:
```

Comment:
Hm... I would expect `templateRef` to be in `required:`..

Reply:
agreed, but even after regenerating these files with your suggestions applied in https://github.com/codeready-toolchain/api/pull/135/commits, I still cannot see it being in the `required`

Reply:
But I do see it in the required now :)